### ISSUE TITLE
Registering ServiceWorker fails with wrong path

### ISSUE THREAD

**Issue by filly82** [parcel-bundler/parcel#1691]
Should be fixed with #1205 and #1206 but still does not work. I've updated parcel to the latest commits (git clone, npm install, npm run build inside the parcel-bundler folder)

## 🤔 Expected Behavior

Parcel should find calls to `navigator.serviceWorker.register` and resolve the given path correctly to a public http url accessible by the browser.

## 😯 Current Behavior

Parcel finds the register() call, but resolves the path wrong. The browser can not load the serviceworker file, therefore the registration fails.

The URL produced by parcel in the example repository is https://localhost:1234/srcserviceworkersw.js

The working URL is https://localhost:1234/src/serviceworker/sw.js (but not preferred)

## 💻 Code Sample

src/index.js
`navigator.serviceWorker.register('./serviceworker/sw.js')`

compiled to
/build/src/serviceworker/sw.js

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | latest commits
| Node             | v8.11.2
| npm/Yarn         | 5.6.0/1.7.0
| Operating System | Win 10 64Bit

**Comment by KeineLimonade** [comment]
Parcel does not have any special treatments for serviceworkers. See parcel-plugin-workbox and parcel-plugin-swprecache

**Comment by filly82** [comment]
@KeineLimonade actually it does, if I understood #398 right ... the produced URL to the service worker is just wrong

as far as I understand parcel-plugin-workbox and parcel-plugin-swprecache, they **always** generate a serviceworker-file. I want my own serviceworker-file to be detected correctly

**Comment by filly82** [comment]
this is the problem, in the compiled src.XXXX.js it is
`navigator.serviceWorker.register('/src**\**sw.js');`
but should be
`navigator.serviceWorker.register('/src**/**sw.js');`

**Comment by KeineLimonade** [comment]
Oh. Ok, thanks for the clarification

**Comment by jessehattabaugh** [comment]
I am experiencing this problem as well. I have the call to `navigator.serviceWorker.register` embedded in my `index.html` in <script> tags, but I've also tried it in an external .js file. I have tried many different permutations of the path: `sw.js`, `./sw.js`, `src/sw.js`, `./src/sw.js`, `/src/sw.js`. But nothing works. The output html file always has the same path, and the `sw.js` never gets output. If I'm getting the path wrong I would expect to see some errors about missing files once and a while but it never complains. 

I kluged around it by creating a `sw.html` file that references `sw.js` and that worked, but it's an ugly hack.
